DataFormats/L1TrackTrigger/src/classes_def.xml

```diff
@@ -51,5 +51,7 @@
   <class name="edm::Wrapper<tt::TTTrackRefMap>"/>
   <class name="TTDTC"/>
   <class name="edm::Wrapper<TTDTC>"/>
+  <class name="std::vector<std::pair<double, double>>"/>
+  <class name="edm::Wrapper<std::vector<std::pair<double, double>>>"/>
```

Comment:
This dictionary should be placed in `DataFormats/WrappedStdDictionaries/src/classes_def.xml`.

Reply:
done